docs/users/step-debugging.md

```diff
@@ -105,7 +105,7 @@ An example configuration:
 ### Visual Studio Code (vscode) Debugging Setup
 
 1. Install the [php-debug](https://marketplace.visualstudio.com/items?itemName=felixfbecker.php-debug) extension.
-2. Update the project's [launch.json] (in `.vscode/launch.json`)  to add "Listen for xdebug" (see [config snippet](snippets/launch.json)).  For more on launch.json, see [vscode docs](https://code.visualstudio.com/docs/editor/debugging#_launch-configurations) )
+2. Update the project's [launch.json] (in `.vscode/launch.json`)  to add "Listen for xdebug" (see [config snippet](snippets/launch.json)).  For more on launch.json, see [vscode docs](https://code.visualstudio.com/docs/editor/debugging#_launch-configurations).
```

Comment:
I believe you're wrong on this one, the right-close paren is closing the clause that starts with "see"

Reply:
I see 4 opening and 5 closing brackets. "see" is already closed `(see [config snippet](snippets/launch.json))`

Reply:
You're right, thanks. Amazing how many spare (or missing) parens and braces there were in here :)